integ-forum/integ-forum-social/src/main/java/org/exoplatform/forum/ext/impl/AnswersSpaceActivityPublisher.java

```diff
@@ -155,11 +155,10 @@ public void saveAnswer(String questionId, Answer answer, boolean isNew) {
       //
       String answerContent = ActivityUtils.processContent(answer.getResponses());
       
-      if (activityId != null) {
+      if (activityId != null && answer.getApprovedAnswers() == true) {
```

Comment:
:one: Ok, I will remove ~~== true~~ in the condition in new commit.

:two: About the small improvement about method's name, I created a task for fix it: https://jira.exoplatform.org/browse/ANS-48

Thx Phi for your comment.